UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common.UnitTests/Helpers/AzureBlobStorageServiceTest.cs

```diff
@@ -88,6 +88,20 @@ public async Task WhenCallStoreSaleCatalogueServiceResponseAsync_ThenReturnsTrue
         #endregion
 
         #region DownloadSalesCatalogueResponse
+
+        [Test]
+        public void WhenScsStorageAccountAccessKeyValueNotfound_ThenReturnKeyNotFoundException()
```

Comment:
Minor Typo: Notfound should be NotFound

Reply:
As per the suggestion i have made changes Notfound should be NotFound.